### PR TITLE
[Feature] Add Ascend support for RoIPool op

### PR DESCRIPTION
**Pull request by momo609**
## Motivation

add roi_pool npu op

## Modification

add roi_pool npu op

### CODE REVIEW

**Review by ckirchhoff2021 — APPROVED** [review]
LGTM

**Comment by zhouzaida** [comment]
Please update unit tests for this NPU op.

File: mmcv/ops/csrc/pytorch/npu/roi_pool_npu.cpp [hunk 1]
```diff
@@ -0,0 +1,34 @@
+#include "pytorch_npu_helper.hpp"
+
+using namespace NPU_NAME_SPACE;
+using namespace std;
+
+Tensor roi_pool_forward_npu(Tensor input, Tensor rois, Tensor output,
```
Comment (zhouzaida) [inline mmcv/ops/csrc/pytorch/npu/roi_pool_npu.cpp:6]: Should `backward` also be supported?

Comment (momo609) [reply mmcv/ops/csrc/pytorch/npu/roi_pool_npu.cpp:6]: backward op is still under development and will be supported later

Comment (grimoire) [reply mmcv/ops/csrc/pytorch/npu/roi_pool_npu.cpp:6]: The return type is different from `roi_pool_forward_impl`

**Review by grimoire — APPROVED** [review]
LGTM after update docs in zh_cn

**Review by zhouzaida — APPROVED** [review]

**Pull request merged**